src/views.js

```diff
@@ -438,13 +438,23 @@ const AttachmentChatboxMessageView = new Lang.Class({
         let thumbnailFactory = Thumbnailer.forSize(GnomeDesktop.DesktopThumbnailSize.LARGE);
         let preview = getPreviewForFile(this.state.path, thumbnailFactory);
         if (preview.thumbnail) {
+            this.attachment_contents.orientation = Gtk.Orientation.VERTICAL;
+
             // Where we are using a thumbnail, use GtkCSSProivder to
             // set the background image
+            let naturalWidth = this.get_preferred_size()[1].width;
+            let minWidth = preview.thumbnail.width < naturalWidth ? naturalWidth : preview.thumbnail.width;
```

Comment:
Hmm, that doesn't seem to work - it seems like the image size doesn't expand as required:

![screen shot 2017-02-14 at 10 33 51 am](https://cloud.githubusercontent.com/assets/775309/22912593/363719e8-f2a1-11e6-854f-fbfb0495a186.png)